harvester/harvester
# fix: change default dashboard UI index to v1.1 release

**IMPORTANT: Please do not create a Pull Request without creating an issue first.**

**Problem:**
The default value of the `ui-index` of v1.1 release should match the v1.1 dashboard release instead of the `latest`.

**Solution:**
Change default dashboard UI index to v1.1 release

**Related Issue:**
https://github.com/harvester/harvester/issues/3078

**Test plan:**
Note: verify using the [v1.1-head.iso](https://releases.rancher.com/harvester/v1.1/harvester-v1.1-amd64.iso)
Single cluster view:
- For either a newly installed Harvester cluster or an upgraded cluster of v1.1-head
- Check the default value of `ui-index` and `ui-plugin-index` is updated accordingly.
- Change `ui-source` setting to `External` and check if the UI can still be loaded successfully (internet access is required.)

Multi-cluster view:
- Import Harvester into the Rancher server
- Go to the Harvester cluster > Advanced > Settings > Change the `ui-source` to external
- Press F12 and check the developer console > Network, ensuring that the UI can still be loaded successfully with the v1.1 plugin version.
<img width="1593" alt="image" src="https://user-images.githubusercontent.com/4569037/200487022-245e80fa-dab7-4bd8-a8c9-37c5154086e5.png">
